extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -919,13 +919,6 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     };
 
     const mutate = () => {
-      if (enter) {
-        Services.ownersForDoc(this.element)./*OK*/ scheduleUnlayout(
-          this.element,
-          this.carousel_
-        );
-      }
-
```

Comment:
I'm not sure that this is desirable. For example, this could cause a video to remain playing. Could you manually test a few cases using `amp-video` and `amp-youtube`? Happy to approve once we verify this.

Reply:
Alan and I are having trouble figuring out what this feature is meant to be supported.

When I added the amp-youtube component, it doesn't appear to trigger the lightbox gallery.

Reply:
Looks like this removed section was introduced in #34630, maybe @kristoferbaxter can provide some context?